src/test/java/gov/usgs/cida/pubs/webservice/LookupMvcServiceBuildDbIT.java

```diff
@@ -170,6 +178,12 @@ public void getOutsideAffiliates() throws Exception {
 
 		assertThat(getRtnAsJSONArray(rtn),
 				sameJSONArrayAs(new JSONArray("[{\"id\":6,\"text\":\"xOutside Affiliation 2\"}]")));
+
+		rtn = performGetRequest(endPoint + "?active=false");
```

Comment:
in this test, endpoint contains "?mimetype=json", so "?active=" should be "&active=" or the mimetype removed from endpoint.